plugins/repositories/core/src/main/javascript/app/components/file/details/details.css

```diff
@@ -1,6 +1,59 @@
-#pentaho_details {
+/*#pentaho_details {
   width: 100%;
   position: absolute;
   top: 0;
   left: 0;
+}*/
+input:disabled {
+  background-color: #f2f2f2;
+  border-color: #e6e6e6;
+  color: #999999;
+}
+
+.text-color-primary {
+  color: #005EAA;
+}
+
+.btn-color-primary {
+  color: white;
+  background-color: #005EAA;
+  border-color: #005EAA;
+}
+
+.btn-color-primary:hover {
+  color: white;
+  background-color: #0F2B5B;
+  border-color: #0F2B5B;
+}
+
+.form-control:focus {
+  border-color: #005EAA;
+  box-shadow: 0 0 0 0.2rem rgba(0,94,170,.25);
+}
+
+#repository-manager-file-details-header {
```

Comment:
Do these comments need to stay? CSS are sent to the client, so they'll waste bandwidth.